### PR TITLE
feature(marginfi-v2-docs & marginfi-v2-ui): change subheading on looper, add section to the arena docs, add ts sdk endpoints

### PR DESCRIPTION
**Pull request by masterzorgon**

### CODE REVIEW

File: apps/marginfi-v2-docs/src/app/faqs/page.mdx [hunk 1]
```diff
@@ -15,6 +15,9 @@ In this guide, we will answer commonly asked questions regarding the marginfi pr
   our [Discord server](https://discord.com/invite/mrgn).
 </Note>
 
+## Can I earn points on The Arena? {{ tag: '', label: '' }}
+No, traders cannot earn points for the activity they conduct on The Arena. Although mrgn. is exploring  Arena is a completely different product to mrgnlend.
```
Comment (chambaz) [inline apps/marginfi-v2-docs/src/app/faqs/page.mdx:19]: couple minor typos, period after mrgn and extra space before Arena

File: apps/marginfi-v2-docs/src/app/the-arena/page.mdx [hunk 2]
```diff
@@ -1,70 +1,110 @@
...
+/>
+
+However, you can earn attractive yields by lending either $BODEN or $USDC into the token bank. Using the $BODEN bank as an example:
+
+* When the trading trend for $BODEN is predominantly short, the APY for lending $BODEN increases.
+* When the trading trend for $BODEN is predominantly long, the APY for lending $USDC increases.
+
+<Note>
+  Lending and borrowing into a similar token bank on <Button href="https://app.marginfi.com/" variant="text"><>mrgnlend</></Button> will not affect the positions you have on a similar token bank on The Arena. **All banks on The Arena are completely isolated from those on mrgnlend**.
+</Note>
```
Comment (chambaz) [inline apps/marginfi-v2-docs/src/app/the-arena/page.mdx:101]: nice

File: packages/marginfi-client-v2/src/client.ts [hunk 3]
```diff
@@ -187,9 +187,9 @@ class MarginfiClient {
         if (response.status === 200) {
           const parsedResponse = await response.json();
           if (!parsedResponse) throw new Error("JSON is mia");
-          const loopupTableString = parsedResponse[config.groupPk.toString()];
+          const lookupTableString = parsedResponse[config.groupPk.toString()];
           if (!parsedResponse) throw new Error("Group not found");
-          addressLookupTableAddresses = [new PublicKey(loopupTableString)];
+          addressLookupTableAddresses = [new PublicKey(lookupTableString)];
```
Comment (chambaz) [inline packages/marginfi-client-v2/src/client.ts:192]: @masterzorgon has this been tested? just want to ensure that variable is not used anywhere else. as this is in the main sdk so high risk.

File: apps/marginfi-v2-docs/src/app/the-arena/page.mdx [hunk 4]
```diff
@@ -1,70 +1,120 @@
...
 
-However, with The Arena, the marginfi team is opening up the ability to create isolated banks to everyone. Now, **you can permissionlessly create isolated banks for any token you want, using any collateral you want** directly through <Button href="https://www.thearena.trade/" variant="text"><>The Arena app</></Button>, the marginfi CLI, or the marginfi TS SDK.  Because permissionless banks are isolated banks, **the trading activity that occurs within any given permissionless bank does not impact marginfi's main liquidity pool**. 
+However, with The Arena, the marginfi team is opening up the ability to create isolated groups to everyone. Now, **you can permissionlessly create isolated banks for any token you want, using any collateral you want** directly through <Button href="https://www.thearena.trade/" variant="text"><>The Arena app</></Button>, the marginfi CLI, or the marginfi TS SDK.  Because permissionless banks are isolated banks, **the trading activity that occurs within any given permissionless bank does not impact marginfi's main liquidity pool**.
```
Comment (k0beLeenders) [inline apps/marginfi-v2-docs/src/app/the-arena/page.mdx:23]: mentioning "you can now create isolated groups" and the sentence after "you can create isolated banks" sounds a bit confusing

File: apps/marginfi-v2-docs/src/app/the-arena/page.mdx [hunk 5]
```diff
@@ -1,70 +1,120 @@
...
 
 <Note>
   This article is not an advertisement of any token, nor a soliciation to purchase or conduct financial activity with any token. This article is purely for educational purposes.
 </Note>
 
 Are you a fan of Iggy Azalea? Permissionlessly create a bank for $MOTHER and open a max-leveraged long position. When you decide you're tired of her music, open a short position. Share the bank you created with others; invite them to make trades with you. You're in the driver's seat.
 
-**Every permissionless bank will belong to its own group**, and the creator of the bank will be assigned as admin over this group. The group will be created along with the permissionless bank.
+**For every token listed on The Arena, an isolated group will be created to hold the token bank (e.g., $BODEN) and the collateral bank (e.g., $USDC)**. The creator of the bank will be assigned as admin over this group.
```
Comment (k0beLeenders) [inline apps/marginfi-v2-docs/src/app/the-arena/page.mdx:31]: Very good explanation

**Review by chambaz — APPROVED** [review]

**Pull request merged**